source/class/qx/tool/cli/commands/Lint.js

```diff
@@ -157,11 +157,6 @@ qx.Class.define("qx.tool.cli.commands.Lint", {
         }
         if (report.errorCount > 0 || report.warningCount > 0) {
           let outputFormat = this.argv.format || "codeframe";
-
-          // If there are too many errors, the pretty formatter is appallingly slow
-          if (report.errorCount + report.warningCount > 150) {
-            outputFormat = "compact";
-          }
```

Comment:
shouldn't this be protected by a command line switch?  EG to always force the output format?  That condition was added specifically because of a massive performance hit (AFACR it made the compiler look like it had hung) so I'm not keen on having it removed.

Reply:
Isn't asking for the specific format a good enough command line switch? Perhaps it should say, if the format is default and the errors are in this range then use the compact format? Even so, if you expect the results back in a certain format, getting a different format is not ideal. Perhaps even force the format if an output file is specified?